pandas/tests/strings/test_strings.py

```diff
@@ -176,6 +177,16 @@ def test_empty_str_methods(any_string_dtype):
     tm.assert_series_equal(empty_str, empty.str.translate(table))
 
 
+@pytest.mark.parametrize("dtype", ["string[pyarrow]"])
+def test_str_contains_compiled_regex_arrow_dtype(dtype):
```

Comment:
The other `.str.contains` tests live in `pandas/tests/strings/test_find_replace.py` (another file in the same directory). Can you move this test there?